server/network.go

```diff
@@ -580,6 +603,8 @@ func (lc *localNetwork) LoadSnapshot(snapshotName string, inPlace bool) error {
 		lc.options.redirectNodesOutput,
 		inPlace,
 		lc.options.walletPrivateKey,
+		lc.options.genesisPath,
```

Comment:
no load snapshot stuff

Reply:
we should save genesis and beaconset into snapshot data